cuebot/src/main/resources/opencue.properties

```diff
@@ -61,6 +61,14 @@ log.frame-log-root.default_os=${CUE_FRAME_LOG_DIR:/shots}
 # are planning to use a folder in the root, use:
 #   - log.frame-log-root.Windows=${S:}
 
+# Loki
+# To enable rqd frame logs to Loki enable this flag and configure the url as shown below. When Loki
+# rqd frame logs are enable all frame logs will be streamed to the loki server instead of using the
+# CUE_FRAME_LOG_DIR filesystem path. Refer to the documentaion on how to configure Loki.
+log.loki.enabled=false
+# This is the base url of the Loki server. If the url is not reachable, rqd will fail running frames.
+log.loki.url=http://localhost/loki/api
```

Comment:
Avoid ambiguous states by removing the enabled option and using url=None as a sign loki is not enabled.

Reply:
`log.loki.enabled` has been removed